contracts/message/apps/examples/MsgTest.sol

```diff
@@ -76,12 +76,8 @@ contract MsgTest is MessageApp {
         return ExecutionStatus.Success;
     }
 
-    function sendMessage(
-        address _receiver,
-        uint64 _dstChainId,
-        bytes calldata _message
-    ) external payable {
-        bytes memory message = abi.encode(nonce, _message);
+    function sendMessage(address _receiver, uint64 _dstChainId) external payable {
+        bytes memory message = abi.encode(nonce, bytes("test101"));
```

Comment:
this may break the sgn e2e tests after contracts bytecode/abi is updated

Reply:
OK. Changed it back to previous version